pkg/connector/org.go

```diff
@@ -199,7 +201,11 @@ func (o *orgResourceType) Grants(
 
 	users, resp, err := o.client.Organizations.ListMembers(ctx, orgName, &opts)
 	if err != nil {
-		return nil, "", nil, fmt.Errorf("github-connectorv2: failed to list org members: %w", err)
+		errMsg := "github-connectorv2: failed to list org members"
+		if isRatelimited(resp) {
+			return nil, "", nil, fmt.Errorf("%s: %w", errMsg, status.Error(codes.Unavailable, "too many requests"))
```

Comment:
can you include the original err message rather than the hardcoded "too many requests"?